sys/auto_init/auto_init.c

```diff
@@ -154,6 +154,11 @@ void auto_init(void)
         extern void auto_init_devfs(void);
         auto_init_devfs();
     }
+    if (IS_USED(MODULE_MOUNTFS)) {
+        LOG_DEBUG("Mounting /.\n");
```

Comment:
I think this would fit well with the automounter. (Not sure whether it should be gated by only MODULE_MOUNTFS or MODULE_VFS_DEFAULTS there; in the latter case it'd need a manual mount in the filesystem example, but then again manual mounting is what that example does.).